src/framework/preferences/js/Panels.js

```diff
@@ -692,6 +692,93 @@ var fluid_3_0_0 = fluid_3_0_0 || {};
         }
     });
 
+    /********************************************************
+     * A base grade for styled radio Button adjuster panels *
+     ********************************************************/
+
+    fluid.defaults("fluid.prefs.panel.styledRadioButtonAdjuster", {
+        gradeNames: ["fluid.prefs.panel"],
+        mergePolicy: {
+            "controlValues.theme": "replace",
```

Comment:
@cindyli we chatted with @dayotte in the fluid-design irc channel and decided to call it themePicker for the time being. 

https://botbot.me/freenode/fluid-design/2017-07-10/?msg=88382236&page=1